.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -11,10 +11,8 @@ Describe what you expected to happen.
 Describe what actually happened that was unexpected.
 
 ### To Reproduce
-Steps to reproduce the behavior:
 
-#### Sample Application
-If applicable, provide a sample application which reproduces the issue.
+An example reproducing the issue in [this repository](https://github.com/microsoft/ApplicationInsights-Java-Repros) would make it easier for the Application Insight maintainers to help you.
```

Comment:
```suggestion
Submitting a PR with an example reproducing the issue in [this repository](https://github.com/microsoft/ApplicationInsights-Java-Repros) would make it easier for the Application Insight maintainers to help you.
```

Reply:
can we mention fork the repo and then submit a PR?

Reply:
@heyam I have pushed an update.